cas/templates/configmap.yaml

```diff
@@ -386,6 +386,12 @@ cas.properties: |-
   cas.callutheran.client.name=${CAS_CALLUTHERAN_CLIENT_NAME:callutheran}
   cas.callutheran.cas.protocol=${CAS_CALLUTHERAN_CAS_PROTOCOL:SAML}
   #
+  # CAS Client: Concordia College
+  cas.cord.login.url=${CAS_CORD_LOGIN_URL:https://bprdeis.cord.edu:8443/cas/login}
```

Comment:
A comment on the port number which appears in the production CAS server URL the institution provided.

The non-standard `8443` SSL port is used in early CAS server deployment. For compatibility reason, SSL port 8443 is also supported for those early deployed CAS applications. However, Apereo / Jasig now strongly recommends to change to standard SSL port as there were problem reported that non-standard port is blocked by some internet providers. It's not a problem now and we can ask them to fix it if their users encounter the issue.